app/src/modules/activity/routes/item.vue

```diff
@@ -85,7 +85,7 @@ export default defineComponent({
 		const error = ref<any>(null);
 
 		const openItemLink = computed(() => {
-			if (!item.value || item.value.collection.startsWith('directus_')) return;
+			if (!item.value || item.value.collection.startsWith('directus_') || item.value.action === 'Delete') return;
```

Comment:
Is this value capitalized? If it is, is it translated?

Reply:
It was indeed translated. TIL. Thanks! 💯